tomviz/python/Recon_ART.py

```diff
@@ -47,6 +49,9 @@ def transform_scalars(self, dataset, Niter=1):
             f[:] = 0
             b = tiltSeries[s, :, :].transpose().flatten()
             for i in range(Niter):
+                self.progress.message = 'Slice No.' + \
+                    str(s + 1) + '/' + str(Nslice) + \
+                    ', iteration No.' + str(i + 1) + '/' + str(Niter)
```

Comment:
Its often simpler todo string substitution, so this expression can be replaced by

```python
 self.progress.message = 'Slice No.%d/%d, iteration No.%d/%d' % (s + 1, Nslice, i + 1, Niter)
```
Is nicely separates the format from the data.